docs/examples_benchmark.md

```diff
@@ -0,0 +1,68 @@
+
+## Table of contents
+
+- [Training Overhead](#real-world-example-benchmarks)
+- [Model Performance](#model-performance)
+- [Scalability with Training](#scalability-with-training)
+
+## Training Overhead
+
+For training overhead, there are two main questions of interest: 1. What is the extra wall-clock time spent on training with FL, compared to train a regular centralized model, but only with 1/#silo of the data. 2. What is the extra computing time (summed from all silos) spent on training with FL, compared to train a regular centralized model with data from all silos combined. The first point is important as it indicates how quickly customers can get their model results, and the second point is essential as it is an indication of the money that customers will spend on computing resource. For this reason, we tested 3 models: "FL" for model trained with FL in 3 silos, "Centralized-1/3" for 1 model with 1/3 data, and "Centralized-1" for 1 model with all data combined. 
+
+### PNEUMONIA
+<p align="center">
+    <img src="./pics/pneumonia_time.jpg" alt="pneumonia training time" width="550"/>
+</p>
+For pneumonia, FL takes only 4% longer wall-clock time than centralized-1/3, and about 5% longer computing time than centralized-1.
+
+### NER
+<p align="center">
+    <img src="./pics/ner_time.jpg" alt="ner training time" width="600"/>
+</p>
+For ner, FL takes only 1.3% longer wall time than centralized-1/3, and only 0.1% longer computing time than centralized-1.
+
+### CCFRAUD
+<p align="center">
+    <img src="./pics/cc_time.jpg" alt="ccfraud training time" width="600"/>
+</p>
+For ccfraud, FL takes 10% longer wall time than centralized model, while about 3% longer computing time than centralized-1.
+
+## Model Performance
+Another important assessing factor for FL is the model performance. Here we also aim at two questions: 1. How does the FL model performance compare to the centralized model trained with only partial data, which is the scenario when FL is not supported and data are confidential and restricted to each region. 2. How does the FL model performance compare to the centralized model trained with data from all silos, which is an ideal situation when all data are eyes-on and could be combined. This is to understand if the distributed-aggregate design of FL has impact on the model performance. For this reason, we tested 3 models: "FL" for model trained with FL in 3 silos, "Centralized-1/3" for 1 model with 1/3 data, and "Centralized-1" for 1 model with all data combined. 
+
```

Comment:
here as well, I would add a highlights of the main findings: that the FL implementation is valid.